plantcv/plantcv/report_size_marker_area.py

```diff
@@ -90,6 +84,7 @@ def report_size_marker_area(img, roi, marker='define', objcolor='dark', thresh_c
             fatal_error('thresh_channel and thresh must be defined in detect mode')
     elif marker.upper() == "DEFINE":
         # Identify contours in the masked image
+        # Are we using a mask to make a new mask here?? lol 
```

Comment:
Looks like yes. I think in this section we could just do `marker_mask = roi_mask` and remove the rest